cmd/minibroker/main.go

```diff
@@ -65,6 +66,8 @@ func main() {
 		"The default namespace for brokers when the request doesn't specify")
 	flag.StringVar(&options.ProvisioningSettingsPath, "provisioningSettings", "",
 		"The path to the YAML file where the optional provisioning settings are stored")
+	flag.StringVar(&options.ClusterDomain, "clusterDomain", "",
+		"The k8s cluster domain - if not set, Minibroker infers from /etc/resolv.conf")
```

Comment:
Needs an example — should this be `cluster.local` or `.cluster.local`?

Reply:
Yes, but it's always good to be clear on what's expected.  Having to re-deploy because there was a dumb mistake sucks.

Reply:
I'm not even exposing this to the chart yet. I'd probably rely on a JSON schema validation if we ever end up exposing this to the chart. http://json-schema.org/understanding-json-schema/reference/regular_expressions.html